sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Can we please make an issue on the Core team to track resetting these?   I trust that we'll catch Tables, but the other libraries aren't owned by us and I do not trust that the change will be noticed or moved back to a package.

Reply:
Yep, happy to.